src/components/select/select.hbs

```diff
@@ -49,8 +49,13 @@
         Validation message here
       </div>
       {{/if}}
-      {{/if}}
     </div>
+    {{/if}}
+    {{#if inline}}
+    {{#if helperText}}
+    <div class="{{@root.prefix}}--form__helper-text">Optional helper text.</div>
```

Comment:
What is the purpose of this element move? Does it work for `v9`, too?

Reply:
this is actually a fix to revert v9 markup to its original state since we did not support inline select with helper text before

Reply:
I can add support for v9 helper text if we want though, since that would address https://github.com/IBM/carbon-components-react/issues/1651

Reply:
Thank you for clarifying @emyarod! The change is fine as long as this PR itself don't introduce a regression.  IBM/carbon-components-react#1651 itself was a regression and we need to see what change introduced the problem.

Reply:
if I understand correctly, we found that the cause was the helper text support being introduced in https://github.com/IBM/carbon-components-react/pull/1218 but without support for inline mode. that being said, I can add inline helper text support to the current PR if that's what we ultimately want

Reply:
Got it - Yes doing it makes sense - Thanks @emyarod!

Reply:
legacy inline select support is now implemented @asudoh